### ISSUE TITLE
fix error "Cannot find module 'lean-qr/extras/svg'"

### ISSUE THREAD

**Issue by moritzpflueger** [davidje13/lean-qr#7]
# Description
I'm trying to add a QR Code in an `vuejs` app. Using the function `toSvgDataURL` will result in an console error:   
`Uncaught Error: Cannot find module 'lean-qr/extras/svg'`

## Usage

```
...

import { generate } from 'lean-qr'
import { toSvgDataURL } from 'lean-qr/extras/svg'

const code = generate(`https://qr.sli.do/${this.eventId}`)

const dataURL = toSvgDataURL(code, {
  on: 'black',
  off: 'transparent',
  padX: 4,
  padY: 4,
  width: null,
  height: null,
  scale: 1
})
console.log('dataURL', dataURL)

...
```

The same error occurrs for all methods relying on the import `lean-qr/extras/svg`

# Full error message in browser console
```
cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/components/widgets/SlidoChat.vue?vue&type=script&lang=js&:10 Uncaught Error: Cannot find module 'lean-qr/extras/svg'
    at webpackMissingModule (cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/components/widgets/SlidoChat.vue?vue&type=script&lang=js&:10:45)
    at eval (cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/components/widgets/SlidoChat.vue?vue&type=script&lang=js&:10:138)
    at ./node_modules/cache-loader/dist/cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/components/widgets/SlidoChat.vue?vue&type=script&lang=js& (app.js:2090:1)
    at __webpack_require__ (app.js:849:30)
    at fn (app.js:151:20)
    at eval (SlidoChat.vue?df27:1:1)
    at ./src/components/widgets/SlidoChat.vue?vue&type=script&lang=js& (app.js:7017:1)
    at __webpack_require__ (app.js:849:30)
    at fn (app.js:151:20)
    at eval (SlidoChat.vue?bbb8:1:1)
```

**Comment by davidje13** [comment]
your import line looks fine, so I have to assume this is an issue with the way Vue is configured (most likely it's not reading the package.json `exports` correctly). Are you able to create a [minimal complete reproduction](https://stackoverflow.com/help/minimal-reproducible-example)? You might also want to check that you're using the latest version of Vue; Webpack added support for wildcard export maps in January 2021, so anything older than WebPack ~5.30 might have this issue.

btw, if possible I'd recommend sticking to `toCanvas` rendering. SVG output is just for when offering a download option to users or if you need to generate an image on the server-side. Maybe that's your use-case, but most of the time it isn't.

**Comment by davidje13** [comment]
I published 1.5.0 which replaces the wildcard exports with explicit exports, which _might_ help you (assuming the issue was some tooling which doesn't understand wildcard exports). Give it a try and let me know, but if it doesn't fix it then I'll need to see an example of it failing to diagnose further.

**Comment by moritzpflueger** [comment]
Thanks for the swift feedback. I did not manage to set up a reproduction. On remote environment it simply works... This makes me think it's something with my local environment. I'll close this issue, until I get the chance to build something that can reproduce the issue.

I'll stick with the canvas option. it's working just fine 👍 

*edit: Thank you for publishing an update. Unfortunately it did not solve the issue for me.
